tools/pip_package/build_pip_package.sh

```diff
@@ -31,6 +32,20 @@ version=${2}
 source_dir=${3}
 blender_dir=${4}
 
+# check if PYTHON_BIN binary is availble
+if ! command -v ${PYTHON_BIN} > /dev/null; then
+    echo "Error: Cannot find ${PYTHON_BIN} binary."
+    exit 1
+fi
+python=$(command -v ${PYTHON_BIN})
```

Comment:
Please address review comments as I mentioned in `src/gen_module.sh` to this file.

Reply:
see comment above, renamed